src/main/kotlin/org/jetbrains/kotlinx/jupyter/messaging/defaultCommHandlers.kt

```diff
@@ -0,0 +1,36 @@
+package org.jetbrains.kotlinx.jupyter.messaging
+
+import kotlinx.serialization.json.JsonObject
+import org.jetbrains.kotlinx.jupyter.ReplForJupyterImpl
+import org.jetbrains.kotlinx.jupyter.api.libraries.Comm
+import org.jetbrains.kotlinx.jupyter.api.libraries.sendData
+import org.jetbrains.kotlinx.jupyter.messaging.ProvidedCommMessages.OPEN_DEBUG_PORT_TARGET
+
+typealias CommMessageProcessCallback = (Comm, JsonObject, ReplForJupyterImpl) -> Unit
+
+interface DefaultCommHandler {
+    val targetId: String
+
+    fun onReceive(): CommMessageProcessCallback
+
+    fun register(repl: ReplForJupyterImpl)
+}
+
+class DebugPortCommHandler : DefaultCommHandler {
+    override val targetId: String
+        get() = OPEN_DEBUG_PORT_TARGET
+
+    override fun onReceive(): CommMessageProcessCallback {
+        return { comm, _, repl ->
+            comm.sendData(OpenDebugPortReply(repl.debugPort))
+        }
+    }
+
+    override fun register(repl: ReplForJupyterImpl) {
```

Comment:
Most likely this method should be deleted... I don't know if we ever would like to override this implementation. Just inline it to the call site

Reply:
Probably yes, anyways, any custom behaviour might be added to api on user demand